docker/Dockerfile

```diff
@@ -14,18 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-FROM centos:7.6.1810
+ARG java_image_tag=8-jdk-focal
+FROM eclipse-temurin:${java_image_tag}
 
 ARG rss_version=0.1.1
 
-RUN yum install -y java-1.8.0-openjdk-devel sshpass openssh-clients vim epel-release bind-utils \
-    supervisor telnet net-tools procps && \
-    yum clean all && rm -rf /var/cache/yum
+USER root
 
-ENV TINI_VERSION v0.19.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /usr/bin/tini
-RUN chmod +x /usr/bin/tini
+RUN set -ex && \
+    apt-get update && \
+    ln -s /lib /lib64 && \
+    apt install -y bash tini sshpass openssh-client vim bind9-utils telnet net-tools procps && \
```

Comment:
always use `apt-get`

Reply:
krb5 releated libs is required since we are going to support HDFS as storage, and let's add busybox